src/renderer/api/endpoints/metrics.api.ts

```diff
@@ -78,10 +78,15 @@ export function normalizeMetrics(metrics: IMetrics, frames = 60): IMetrics {
       result.forEach(res => {
         if (!res.values || !res.values.length) return;
 
+        let now = moment().unix();
+
         while (res.values.length < frames) {
-          const timestamp = moment.unix(res.values[0][0]).subtract(1, "minute").unix();
+          const timestamp = moment.unix(now).subtract(1, "minute").unix();
```

Comment:
Also, why is it better to `unshift` values from "now" instead of the first time position?

Reply:
First time position might be "X minutes ago", we need to normalize these metrics from "now".

Reply:
So why not instead of using `now` we just take `res.values[0][0]` and start subtracting from there?

Reply:
> So why not instead of using `now` we just take `res.values[0][0]` and start subtracting from there?

Now it uses `res.values[0][0]` so that we don't hit into clock sync issues and uses both subtract & add to fill gaps both ways.